jekyll/_cci2/templates/template-overview.adoc

```diff
@@ -0,0 +1,153 @@
+---
+# These platform content tags control the tags that appear at the top of the docs page. Use them to show the reader which CircleCI platforms support the feature or task you are writing about
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Page title
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+////
+Some notes on attributes
+
+:page-liquid: - ensures that all liquid tags are processed before rendering the content. More info here: https://github.com/asciidoctor/jekyll-asciidoc/blob/89b8f55f5312e4a0f1bca496bd9217d53d5b21dd/docs/modules/ROOT/pages/liquid.adoc
+
+:icons: font - enables the use of font awesome icons https://docs.asciidoctor.org/asciidoc/latest/macros/icons-font/
+
+:experimental: allows access to asciidoc macros, more info here: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/
+
+////
+
+This template will help you create a new docs page to provide conceptual information about a feature or task in CircleCI. To use this template make a copy and place it in the `_cci2` directory. The filename should match the page title. Look at the source file for this page template link:https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/template/template-overview.adoc?plain=1[here]. There is additional detail in the comments.
+
+// If possible display the use cases in a bullet list:
+
+The opening paragraph should be used to succinctly explain to the reader what value the feature/task provides and what use cases it supports. Content templates help you to:
+
+* Develop new content quickly
```

Comment:
I think this could be broken out further and rearranged into -

```suggestion
// Opening paragraph/s (does not require a heading)

This template will help you create a new docs page to provide **conceptual** information about a feature or task in CircleCI. Content templates help you to: ...

* Develop new content quickly
* Ensure your page conforms to the style guide

To use this template, make a copy and place it in the `_cci2` directory. The filename should match the page title. Look at the source file for this page template link:https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/template/template-overview.adoc?plain=1[here]. There is additional detail in the comments.

The opening paragraphs should be used to **succinctly** explain to the reader what value the feature/task provides and what use cases it supports.

// * If possible briefly mention some use cases in a bullet list:
```